bedrock/firefox/tests/test_views.py

```diff
@@ -135,6 +136,7 @@ def test_variant_android_invalid_test_value(self):
                                                lang='en-US')
 
 
+@override_settings(DEV=False)
```

Comment:
This is here because this test failed for me locally when I had `DEV=True` in my `.env` file.